validate.sh

```diff
@@ -13,7 +13,7 @@
 HC=ghc
 CABAL=cabal
 CABALPLAN=cabal-plan
-JOBS=4
+JOBS=12
```

Comment:
This is bold! You realize that it won't affect the CI, right? 'Cause validate.yml sets its own `-j` and passes it to this script.

Reply:
ergh, yeah it's a remnant from my local development. :)